integration-cli/docker_cli_external_graphdriver_unix_test.go

```diff
@@ -102,9 +102,7 @@ func (s *DockerExternalGraphdriverSuite) SetUpSuite(c *check.C) {
 	base, err := ioutil.TempDir("", "external-graph-test")
 	c.Assert(err, check.IsNil)
 	vfsProto, err := vfs.Init(base, []string{}, nil, nil)
-	if err != nil {
-		c.Fatalf("error initializing graph driver: %v", err)
-	}
+	c.Assert(err, check.IsNil, check.Commentf("error initializing graph driver: %v", err))
```

Comment:
You can omit `err` in the comment, it will be printed out anyway :wink: 

```go
c.Assert(err, check.IsNil, check.Commentf("error initializing graph driver"))
```